### ISSUE TITLE
EMT solars don't work with gt transformers. 

### ISSUE THREAD

**Issue by benstuwie** [GTNewHorizons/NewHorizons#2429]
#### Which modpack version are you using?
2.0.2.6
#### What do you suggest instead/what changes do you propose?
The electromagnetic tools solars use Ic2 power, but unlike a lot of other things that do, they don't work with gt transformers. The water/wind turbines are also ic2 but you can put a gt transformer directly on them to take the power out, unlike the emt solars. 
It's not really a bug, but it would make things more convenient if you could put a gt transformer on them to keep all the power GT.

**Comment by Dream-Master** [comment]
Yes you need an ic2 transformer first I guess. @mitchej123 are you able to add electro magic tools support. We forked the project allready. Thanks

**Comment by LOKKO12** [comment]
EMT Solar -> IC2 Transfromer -> GT Transformer -> Cable/Battery Buffer 

TBH i recommend using the "normal" solars instead of EMT ones, they are cheaper.

**Comment by bartimaeusnek** [comment]
Fixed. they dont need any transformer now. (EMT 1.2.7.1)
